test/v1_2_0/base/EscrowBase.sol

```diff
@@ -381,7 +381,8 @@ contract EscrowBase is
         address _dao,
         address _clock
     ) public returns (LinearIncreasingCurve) {
-        LinearIncreasingCurve impl = new LinearIncreasingCurve();
+        (int256[3] memory coefficients, uint256 maxEpoch) = CurveConstantLib.getCoefficients();
```

Comment:
I see what you've done here but doesn't this defeat the point of having the variables - now we still need a new CurveConstantLib for new deploys?